osquery/tables/system/darwin/process_open_descriptors.cpp

```diff
@@ -65,7 +65,49 @@ void parseNetworkSocket(const struct socket_fdinfo socket_info, Row& r) {
   // Set socket protocol.
   const struct in_sockinfo* in = nullptr;
   if (socket_info.psi.soi_kind == SOCKINFO_TCP) {
-    in = &socket_info.psi.soi_proto.pri_tcp.tcpsi_ini;
+    const struct tcp_sockinfo* tcp_in = &socket_info.psi.soi_proto.pri_tcp;
+    in = &tcp_in->tcpsi_ini;
+    switch (tcp_in->tcpsi_state) {
+    case TSI_S_CLOSED:
+        r["state"] = "CLOSE";
```

Comment:
CLOSED

Reply:
Fix it here too? https://github.com/facebook/osquery/blob/master/osquery/filesystem/linux/proc.h#L70

I was trying to be consistent

Reply:
Good point. Just checked netstat too and it also uses CLOSE, so leave it a this.